app/templates/0.4.0/_bower.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "<%= slugifiedAppName %>",
+  "version": "0.0.0",
+  "description": "<%= appDescription %>",
+  "dependencies": {
+    "bootstrap": "~3",
+    "angular": "~1.3",
+    "angular-resource": "~1.3",
+    "angular-animate": "~1.3",
+    "angular-mocks": "~1.3",
+    "angular-bootstrap": "~0.13",
+    "angular-ui-utils": "bower",
+    "angular-ui-router": "~0.2",
+    "angular-file-upload": "1.1.5"
+  },
+  "resolutions": {
```

Comment:
This was copied from the MEAN repo. I think there are some instances where the install will fail in noninteractive mode when there's certain dependency conflicts.